spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/ServiceInstanceListSupplierBuilder.java

```diff
@@ -248,6 +248,13 @@ public ServiceInstanceListSupplierBuilder withHints() {
 		return this;
 	}
 
+	public ServiceInstanceListSupplierBuilder with(DelegateCreator delegateCreator) {
```

Comment:
Please add javadoc for method.

Reply:
OK, I will add it soon.

Reply:
@OlgaMaciaszek Please check it, meet the requirements?